Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2006-05-30
Initial Package Version: 2.12r
Origin: Jim Gifford
Upstream Status: Sent
Description: Fixes the missing delcaration of R_OK in swapon.c build
 
diff -Naur util-linux-2.12r.orig/mount/swapon.c util-linux-2.12r/mount/swapon.c
--- util-linux-2.12r.orig/mount/swapon.c	2004-12-22 01:50:19.000000000 -0800
+++ util-linux-2.12r/mount/swapon.c	2006-05-30 00:15:28.979305328 -0700
@@ -11,6 +11,7 @@
 #include <mntent.h>
 #include <errno.h>
 #include <sys/stat.h>
+#include <fcntl.h>
 #include "xmalloc.h"
 #include "swap_constants.h"
 #include "swapargs.h"
